sirmordred/task_enrich.py

```diff
@@ -255,7 +256,8 @@ def __studies(self):
               [study for study in studies]))
 
         studies_args = self.__load_studies()
-        do_studies(enrich_backend, studies_args)
+
+        do_studies(ocean_backend, enrich_backend, studies_args)
```

Comment:
do we need to change do_studies method?

Reply:
I would say so, in this way elk can access the information of the ocean backend (and in particular the elasticsearch url)